src/firmware/boards/legacy_dongle_stm32f4/normal.c

```diff
@@ -27,14 +27,14 @@
 #include "radio_config.h"
 
 /**
- * \brief The number of robots.
+ * The maximum expected drive packet size
```

Comment:
Maybe throw in a 'bytes' here

Reply:
changed to `The maximum expected drive packet size, in bytes`